pkgs/applications/editors/jetbrains/linux.nix

```diff
@@ -99,7 +103,9 @@ with stdenv; lib.makeOverridable mkDerivation (rec {
     jdk=${jdk.home}
     item=${desktopItem}
 
-    makeWrapper "$out/$pname/bin/${loName}.sh" "$out/bin/${pname}" \
+    mv $out/$pname/bin/${loName}.sh $out/$pname/bin/${loName}_wrapped.sh
+
+    makeWrapper "$out/$pname/bin/${loName}_wrapped.sh" "$out/$pname/bin/${loName}.sh" \
```

Comment:
I think this can be replaced with [`wrapProgram`](https://nixos.org/manual/nixpkgs/unstable/#fun-wrapProgram)

Reply:
I didn't know about wrapProgram. That's very useful! I learned a lot during this PR! Thank you!